terraform/cyhy_feeds.tf

```diff
@@ -0,0 +1,64 @@
+# The feeds EC2 instance
+data "aws_ami" "cyhy_feeds" {
+  filter {
+    name = "name"
+    values = [
+      "cyhy-feeds-hvm-*-x86_64-ebs"
+    ]
+  }
+   filter {
+    name = "virtualization-type"
+    values = ["hvm"]
+  }
+   filter {
+    name = "root-device-type"
+    values = ["ebs"]
+  }
+   owners = ["${data.aws_caller_identity.current.account_id}"] # This is us
+  most_recent = true
+}
+ resource "aws_instance" "cyhy_feeds" {
+  ami = "${data.aws_ami.cyhy_feeds.id}"
+<<<<<<< HEAD
+  instance_type = "${local.production_workspace ? "r5.large" : "t2.micro"}"
+=======
+  instance_type = "t2.micro"
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+  # ebs_optimized = true
+  availability_zone = "${var.aws_region}${var.aws_availability_zone}"
+   # This is the private subnet
+  subnet_id = "${aws_subnet.cyhy_private_subnet.id}"
+<<<<<<< HEAD
+=======
+  private_ip = "${cidrhost(aws_subnet.cyhy_private_subnet.cidr_block, local.the_feeds)}"
```

Comment:
We no longer assign fixed IPs like this.  See the most recent code and update accordingly.